easybuild/easyconfigs/c/CHASE/CHASE-v1.eb

```diff
@@ -0,0 +1,28 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Tarball"
+
+name = 'CHASE'
+version = 'v1'
```

Comment:
@boegel Hmm...I went to their website, but it doesn't say the date of update. It just provides a link to download. What would you recommend?